.github/dependabot.yml

```diff
@@ -0,0 +1,8 @@
+version: 2
+updates:
+  - package-ecosystem: "npm"
+    versioning-strategy: lockfile-only
+    directory: "/"
+    schedule:
+      interval: "daily"
+    labels: ["dependencies"]
```

Comment:
Nit: missing newline

Reply:
Good catch! `redhat.vscode-yaml` now takes care of that for me :)